pom.xml

```
@@ -101,6 +114,20 @@
             <artifactId>virtual-schema-shared-integration-tests</artifactId>
             <version>3.0.0</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <!-- Workaround for https://github.com/exasol/virtual-schema-shared-integration-tests/issues/43 -->
+                    <groupId>org.junit.jupiter</groupId>
+                    <artifactId>junit-jupiter</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

```suggestion
            <version>3.0.1</version>
            <scope>test</scope>
        </dependency>
```
This is fixed in the latest version :-)